sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.aest.controller.getApplicationDetails.e2e-spec.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../testHelpers";
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { EducationProgramOffering, OfferingIntensity } from "@sims/sims-db";
+import { getUserFullName } from "../../../utilities";
+import {
+  addDays,
+  getDateOnlyFormat,
+  getISODateOnlyString,
+} from "@sims/utilities";
+
+describe("ApplicationAESTController(e2e)-getApplicationDetails", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should get the student application details.", async () => {
+    // Arrange
+    const offeringInitialValues = {
+      studyStartDate: getISODateOnlyString(addDays(-10)),
+      studyEndDate: getISODateOnlyString(addDays(10)),
+      offeringIntensity: OfferingIntensity.fullTime,
+    } as EducationProgramOffering;
+
+    const student = await saveFakeStudent(db.dataSource);
+
+    const application = await saveFakeApplication(
+      db.dataSource,
+      { student: student },
+      {
+        offeringInitialValues: offeringInitialValues,
+      },
+    );
+
+    await db.application.save(application);
+
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+
+    const endpoint = `/aest/application/${application.id}`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        data: {},
+        id: application.id,
+        applicationStatus: application.applicationStatus,
+        applicationNumber: application.applicationNumber,
+        applicationFormName: "SFAA2022-23",
+        applicationProgramYearID: application.programYearId,
+        studentFullName: getUserFullName(application.student.user),
+        applicationOfferingIntensity: offeringInitialValues.offeringIntensity,
+        applicationStartDate: getDateOnlyFormat(
+          offeringInitialValues.studyStartDate,
+        ),
+        applicationEndDate: getDateOnlyFormat(
+          offeringInitialValues.studyEndDate,
+        ),
+        applicationInstitutionName:
+          application.location.institution.legalOperatingName,
+      });
+  });
+
+  it("Should get the student application details without dynamic data when loadDynamicData is false.", async () => {
+    // Arrange
+    const offeringInitialValues = {
+      studyStartDate: getISODateOnlyString(addDays(-10)),
+      studyEndDate: getISODateOnlyString(addDays(10)),
+      offeringIntensity: OfferingIntensity.fullTime,
+    } as EducationProgramOffering;
+
+    const student = await saveFakeStudent(db.dataSource);
+
+    const application = await saveFakeApplication(
```

Comment:
When creating fake application, make sure that some dynamic data is added (which is not returned using `loadDynamicData` = false.)